setup.py

```diff
@@ -1,7 +1,8 @@
 from setuptools import find_packages, setup
 
 install_requires = [
-    "wagtail>=2.7,<3",
+    "wagtail>=2.15,<4.0",
+    "django>=3.0,<5.0",
```

Comment:
Let's limit this to 3.2, the current LTS version, with appropriate changes elsewhere here.
```suggestion
    "django>=3.2,<5.0",
```

Reply:
I have made updates to this file to include some refinements.